tests/labhub_test.py

```diff
@@ -22,6 +26,12 @@
 class TestLabHub(unittest.TestCase):
 
     def setUp(self):
+        labhub_info_path = (Path(__file__)
```

Comment:
This should be put into a helper somewhere.

Reply:
I shifted this to `CoroboTestHelper` class in the commit I am about to push once https://github.com/coala/corobo/pull/573/ gets merged. But I'll move this into `helpers.py` for now.

Reply:
Doesn't make much sense because this is not generic and needed in all plugins using the helper. Specific to plugin stays specific to test cases, no?